server/controllers/index.js

```diff
@@ -72,72 +82,77 @@ router.get('/api/start', auth, (req, res) => {
         res.sendStatus(200);
     } catch (err) {
         console.log(err);
-        res.status(400).json({message: err.message});
+        res.status(400).json({ message: err.message });
     }
 });
 
 /* End the game */
 router.get('/api/end', auth, (req, res) => {
     const { roomId } = req.session;
     try {
-        const gameState = Room.getById(req.session.roomId).gameState;
-        gameState.endGame();
-        Room.deleteById(roomId);
-        closeRoom(roomId);
-        res.sendStatus(200);
+        const gameState = Room.getById(roomId).gameState;        
+        gameState.getPlayers().forEach(player => {
+            db.updateScore(player.id)
```

Comment:
How is it updating the score if not given the score?

Reply:
good point I'm updating it with the prev score, let me fix that